### ISSUE TITLE
Is the attribution "lr" controlling learning rate?

### ISSUE THREAD

**Issue by Caoang327** [facebookresearch/synsin#34]
Hey. 
Thanks for the great project and code. 
I am running the code by myself and I just wonder does the "lr" in the train.sh actually control anything? 
In the model/base_model.py, the learning rates for G and D are specified by lr_g and lr_d (line 21),
and it seems that lr_g and lr_d are default values that are different from the reported number in the paper. 
I just wonder is my understanding correct?
